game.cpp

```diff
@@ -3,6 +3,14 @@
 
 #include "game.h"
 #include "raycaster.h"
+#include "raycaster_data.h"
+
+inline float distance(float x1, float y1, float x2, float y2)
```

Comment:
Add `static` as the qualifier, so that compilers can perform better optimizations.